docs/setup/hybrid.md

```diff
@@ -29,6 +29,6 @@ redis:
 
 ## Running
 
-1. To start Postgres and Redis: `docker-compose -f docker-compose-deps.yml up`
+1. To start Postgres and Redis: `bin/deps`
```

Comment:
Another option is we just have them run the docker command here instead of creating bin/deps. 🤔